src/main/java/org/folio/inventory/dataimport/handlers/actions/CreateInstanceEventHandler.java

```diff
@@ -54,6 +56,8 @@
   private MappingMetadataCache mappingMetadataCache;
   private IdStorageService idStorageService;
 
+  private Instance duplicatedInstance = null;
```

Comment:
Is this field used?

Reply:
@SerhiiNosko Thank you! Forget remove it after local debugging.